### PR TITLE
Fix "*.exe.config" file usage in nuspec files. Fixes #2616.

### LINKED ISSUE #2616: OrleansHost.exe.config is not placed in output folder in consuming project
When investigating the #2614 issue it turned out that OrleansHost.exe.config is not getting copied when the user builds a project which references the ```Microsoft.Orleans.OrleansHost``` package.

I think that this is an error in the given nuspec file and we need to use ```contentFiles```.

### PR DESCRIPTION
**Pull request by attilah**
- Modify Nuget packages to copy executable and other tool files to output directory.
- Fix file order in nuspec files (cosmetic).

### CODE REVIEW

File: src/OrleansManager/Microsoft.Orleans.OrleansManager.targets [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="CopyOrleansManagerTools" BeforeTargets="AfterBuild">
+    <ItemGroup>
+      <FilesToCopy Include="$(MSBuildThisFileDirectory)..\tools\OrleansManager.exe.config" />
+      <FilesToCopy Include="$(MSBuildThisFileDirectory)..\tools\ClientConfiguration.xml" />
```
Comment (ReubenBond) [inline src/OrleansManager/Microsoft.Orleans.OrleansManager.targets]: Is there any chance that this config will override user config?

Comment (attilah) [inline src/OrleansManager/Microsoft.Orleans.OrleansManager.targets]: I don't know. We only supply this in a Nuget for users who want no hassle usage, like OrleansHost.exe. The Nuget is designed to run from Output directory, but Nuget was not copying that file.
With the current setup the user can overwrite this file in an AfterBuild or later target if needed, but I don't think that anyone runs it from output directory in a live environment ;-)

Comment (ReubenBond) [inline src/OrleansManager/Microsoft.Orleans.OrleansManager.targets]: I wonder if a conditional copy is doable

Comment (attilah) [inline src/OrleansManager/Microsoft.Orleans.OrleansManager.targets]: We can do that too. I'll talk with @sergeybykov what is he thinking about this.

Comment (ReubenBond) [inline src/OrleansManager/Microsoft.Orleans.OrleansManager.targets]: Sounds good

**Pull request merged**